packages/browser/src/index.test.ts

```diff
@@ -166,13 +166,16 @@ describe('LogtoClient', () => {
 
     test('tokens should be set after calling signIn and handleSignInCallback successfully', async () => {
       const logtoClient = new LogtoClient({ endpoint, clientId }, requester);
+      expect(() => logtoClient.hasSignInRedirected(redirectUri)).toMatchError(
+        new LogtoClientError('sign_in_session.not_found')
+      );
```

Comment:
should extract to a separate test case? As the original test descriptions said `handleSignInCallback successfully`

Reply:
I can do that, but that would end up with many duplications. Is it necessary?
Or maybe I can slightly update the description

Reply:
Done